### PR TITLE
Update FeedParser

### PR DESCRIPTION
**Pull request by flooie**
PR Resolves a conflict between juriscraper and CL over feed parser.

This PR does two things.  

It bumps juriscraper to the most recenter version.
It bumps feed parser in courtlistener to 6.0.8 to match juriscraper.

It updates a selenium test that was failing... The test required an implicit wait to 
load the modal, which was not occurring in time and adds an assertion to verify
that the modal has loaded.  The Create an Alert text was actually not helping
validate that the modal had loaded (I believe).  Instead I added a test for the class
name to validate the modal for creating an alert has loaded.

### CODE REVIEW

File: poetry.lock [hunk 1]
```diff
@@ -1606,6 +1608,14 @@ sanic = ["sanic (>=0.8)"]
 sqlalchemy = ["sqlalchemy (>=1.2)"]
 tornado = ["tornado (>=5)"]
 
+[[package]]
+name = "sgmllib3k"
```
Comment (mlissner) [inline poetry.lock:1612]: Any idea what's going on here? The package's description is dire. Is this part of feedparser? I hope not.

**Review by mlissner — COMMENTED** [review]
One q.

File: poetry.lock (hunk 1 above)
Comment (satsuki-chan) [reply poetry.lock:1612]: It is a `feedparser 6.0.8` dependency:

https://github.com/kurtmckee/feedparser/blob/develop/pyproject.toml#L21-L23

Comment (flooie) [reply poetry.lock:1612]: always one q.  yes its dire sounding but... that's never stopped us before right?

**Comment by mlissner** [comment]
And so we welcome an already dire dependency into our warm embrace.

**Pull request merged**